### ISSUE TITLE
V3.1.12 no windows build archive + V3.1.7 for windows 4k fullscreen bug

### ISSUE THREAD

**Issue by roman80k** [projectM-visualizer/projectm#555]
**Describe the bug**
1. There is no latest version build for Windows available
2. Latest Windows build on 4k monitor fullscreen leaves visible space at the bottom, around 50-100px.

**Screenshots**
https://snipboard.io/vVophK.jpg

**Additional context**
Scale in Windows settings is set to 200%, just in case

**Comment by kblaschke** [comment]
Even v3.1.12 is outdated, as since then, a lot of development work has happened. There were numerous bugs with fullscreen mode and the built-in menu which are fixed in the current source, but there is no new release yet as we switched to a new build system and the process isn't complete yet.

If you want to try out the current state of work, I'll be releasing a ready-to-use package made from the automated GitHub Actions builds from time to time, which you can find in my development fork of this repository:

https://github.com/kblaschke/projectm/releases

There you'll find a ZIP I've created a few days ago, including configuration instructions. Please give it a try and report if this fixed your above issues.

**Comment by roman80k** [comment]
Thanks man, I see.

Your version is fine: full screen is ok, so that issue is fixed.

But it seems to ignore settings file. I've copied it to %USERPROFILE%, I see it has been parsed - all comments were stripped down.

- I see "Fullscreen = false" there, but anyway app launches in full screen.
- FPS is set to 35 by default, but no matter what I set, I am getting 64 FPS (I have 60Hz set)
- Shuffle also seems to be ignored, every time I relaunch app - it seems that presets go in one order.

- Also mouse cursor is not hiding automatically. And as an end-user - I also would like to have some basic control with mouse, for example: double click switches app to/from full screen...


But anyway yes, 3.1.13 is much more stable.

**Comment by kblaschke** [comment]
Yes, the `Fullscreen` parameter is currently being ignored in the SDL frontend and hiding the cursor is also not implemented.

Besides other improvements to this app, these things are on my long list of fixes before the actual 4.0 release. Really need to add a project with tasks here to keep track.
